### PR TITLE
better design for github-contributors, add mapbox key env

### PR DESCRIPTION
**Pull request by georgios-uber**
allow additional link positions, fix homepage...

### CODE REVIEW

File: modules/gatsby/src/components/github/github-contributors.jsx [hunk 1]
```diff
@@ -28,19 +57,20 @@ export default class GithubContributors extends Component {
 
   render() {
     const {response} = this.state;
```
Comment (ibgreen) [inline modules/gatsby/src/components/github/github-contributors.jsx]: Just having `response` in state looks a bit unfinished. Can't the part of code that loads do the data verification and store `contributors` in state?

File: modules/gatsby/src/components/site-query.jsx [hunk 2]
```diff
@@ -41,6 +41,7 @@ const QUERY = graphql`
         ADDITIONAL_LINKS {
           name
           href
+          index
```
Comment (ibgreen) [inline modules/gatsby/src/components/site-query.jsx]: If you are adding features, please document them!

Comment (georgios-uber) [reply modules/gatsby/src/components/site-query.jsx]: Done

File: modules/gatsby/src/components/github/github-contributors.jsx (hunk 1 above)
Comment (georgios-uber) [reply modules/gatsby/src/components/github/github-contributors.jsx]: Probably in another diff.

**Pull request merged**